src/App.jsx

```diff
@@ -1,10 +1,106 @@
+import axios from 'axios'
 import './App.css';
+import {useState} from "react";
+import globe from "./assets/spinning-globe.jpg"
+import worldmap from "./assets/world_map.png"
+import regionColorNames from "./helpers/regionColorHelper.js";
+import sortPopulationHelper from "./helpers/sortPopulationHelper.js";
 
 function App() {
 
+    const [countries, setCountries] = useState("")
```

Comment:
`countries` gaat nooit een andere string worden. Ik zou zelf gewoon lege array als default waarde pakken. Dan worden je checks ook wat simpeler
```suggestion
    const [countries, setCountries] = useState([]);
```